content/400-reference/200-api-reference/250-error-reference.mdx

```diff
@@ -122,45 +116,6 @@ Prisma Client throws a `PrismaClientValidationError` exception if validation fai
 
 Possible errors:
 
-- "Argument `{}` is missing."
-- "Function `{}` takes {} arguments, but received {}."
-- "Argument `{}` is missing in attribute `@{}`."
-- "Argument `{}` is missing in data source block `{}`."
-- "Argument `{}` is missing in generator block `{}`."
-- "Error parsing attribute `@{}`: {}"
-- "Attribute `@{}` is defined twice."
-- "The model with database name `{}` could not be defined because another model with this name exists: `{}`"
-- "`{}` is a reserved scalar type name and can not be used."
-- "The {} `{}` cannot be defined because a {} with that name already exists."
-- "Key `{}` is already defined in {}."
-- "Argument `{}` is already specified as unnamed argument."
-- "Argument `{}` is already specified."
-- "No such argument.""
-- "Field `{}` is already defined on model `{}`."
-- "Field `{}` in model `{}` can't be a list. The current connector does not support lists of primitive types."
-- "The index name `{}` is declared multiple times. With the current connector index names have to be globally unique."
-- "Value `{}` is already defined on enum `{}`."
-- "Attribute not known: `@{}`."
-- "Function not known: `{}`."
-- "Datasource provider not known: `{}`."
-- "shadowDatabaseUrl is the same as url for datasource `{}`. Please specify a different database as shadow database."
-- "The preview feature `{}` is not known. Expected one of: {}"
-- "`{}` is not a valid value for {}."
-- "Type `{}` is neither a built-in type, nor refers to another model, custom type, or enum."
-- "Type `{}` is not a built-in type."
-- "Unexpected token. Expected one of: {}"
-- "{}"
-- "{}"
-- "{}"
-- "Environment variable not found: {}."
-- "Expected a {} value, but received {} value `{}`."
-- "Expected a {} value, but failed while parsing `{}`: {}."
-- "Error validating model `{}`: {}"
-- "Error validating field `{}` in model `{}`: {}"
-- "Error validating datasource `{datasource}`: {message}""
-- "Error validating enum `{}`: {}"
-- "Error validating: {}"
```

Comment:
These were probably collected manually and might currently be skipped by the script. 
Should def not be removed, but maybe the script adapted to get them automatically.